core/include/traccc/edm/cluster.hpp

```diff
@@ -9,31 +9,46 @@
 
 // Project include(s).
 #include "traccc/edm/cell.hpp"
+#include "traccc/geometry/pixel_data.hpp"
 
 // System include(s).
 #include <functional>
 #include <vector>
 
 namespace traccc {
 
-using position_estimation = std::function<vector2(channel_id, channel_id)>;
-using signal_modeling = std::function<scalar(scalar)>;
+/// Function for signal modelling
+inline scalar signal_cell_modelling(scalar signal_in) {
```

Comment:
This function will need to receive a cluster/module ID as well. Right now the "signal" from the detectors is not processed in any shape or form. But in principle there was a reason why this function was part of the "ID object" before. Because it could very much depend on the detector module, or even the specific cluster, how significant the same "digitised counts" from the detector would be treated as.

Reply:
An unused module ID parameter won't compile in a full debug mode. Is there a way to work around it?

Reply:
Done.